src/main/target/common_fc_pre.h

```diff
@@ -118,6 +117,7 @@
 #define USE_RX_MSP
 #define USE_SERIALRX_JETIEXBUS
 #define USE_SENSOR_NAMES
+#define USE_SERVOS
```

Comment:
Can we disable MSP over SmartPort / CRSF instead of servos for F1? I feel that servos are a big trade-off for F1 boards.

Reply:
working this now